tests/ut/ir/test_compiled_program.py

```
@@ -515,5 +516,337 @@ def test_device_tensor_dtype_mismatch_rejected_early(self, tmp_path):
+        oa_helper.assert_called_once_with([a, b, c])
+
+    def test_build_orch_args_return_style_allocates_output(self, tmp_path):
+        """Return-style: only inputs passed; output auto-allocated at output_indices."""
+        prog = _make_program_with_orchestration(has_return=True)
+        cp = CompiledProgram(prog, str(tmp_path))
+        a = torch.zeros(128, 128)
+        b = torch.zeros(128, 128)
+
+        with patch("pypto.runtime.runner._coerced_to_orch_args") as oa_helper:
+            oa_helper.return_value = "fake_orch_args"
+            orch_args, coerced, return_style = cp.build_orch_args(a, b)
```

Ruff (0.15.13)

[warning] Unpacked variable `orch_args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)